styles/style_skills.css

```diff
@@ -0,0 +1,55 @@
+body {
+  font-family: "Raleway", sans-serif;
+}
+
+section {
+  width: 60vw;
+  height: auto;
+  margin: auto;
+}
```

Comment:
Fajnie by było te style zastosować do każdej sekcji, żeby nie powtarzać kodu, jedną z możliwości na pewno by było stworzenie nowego pliku css gdzie byśmy dawali style globalne typu h1,h2, ażeby a każdej sekcji wyglądało to identycznie

Reply:
Racja. Przeniosłam do nowego pliku i podlinkowałam w index.html.

Reply:
Ten plik też zaimportujmy tylko do tego naszego main.css, to w zupełności wystarczy